.github/workflows/ci.yml

```diff
@@ -92,6 +93,14 @@ jobs:
       run: |
         go test -v ./...
 
+    - name: static-check
```

Comment:
I'd like to have this earlier, or potentially running in parallel in it's own job.

If you rename it to be "lint" to match the golangci-lint job, then the branch protection rule doesn't need to be updated.